core/src/main/kotlin/org/evomaster/core/problem/graphql/builder/StateBuilder.kt

```diff
@@ -0,0 +1,1146 @@
+                                typeName = elementInTypes.name,
+
+                                enumValues = enumElement,
+                                unionTypes = listOf(),
+                                interfaceTypes = listOf()
+                            )
+                        )
+
+                    }
+                    __TypeKind.UNION -> {
+                        val unionElement: MutableList<String> =
+                            collectUnionElementsInTable(schemaObj, elementInfields.type.name)
+                        state.tables.add(
+                            Table(
+                                fieldName = fieldName,
+                                isFieldNameWithArgs = isFieldNameWithArgs,
+
+                                isKindOfFieldTypeOptional = true,
+
+                                kindOfFieldType = kind0.toString(),
+                                fieldType = elementInfields.type.name,
+                                typeName = elementInTypes.name,
+
+                                enumValues = listOf(),
+                                unionTypes = unionElement,
+                                interfaceTypes = listOf()
+                            )
+                        )
+                    }
+                    __TypeKind.INTERFACE -> {
+                        val interfaceElement: MutableList<String> =
+                            collectInterfaceElementsInTable(schemaObj, elementInfields.type.name)
+                        state.tables.add(
+                            Table(
+                                fieldName = fieldName,
+                                isFieldNameWithArgs = isFieldNameWithArgs,
+
+                                isKindOfFieldTypeOptional = true,
+
+                                kindOfFieldType = kind0.toString(),
+                                fieldType = elementInfields.type.name,
+                                typeName = elementInTypes.name,
+
+                                enumValues = listOf(),
+                                unionTypes = listOf(),
+                                interfaceTypes = interfaceElement
+                            )
+                        )
+                    }
+                    else -> {
+                        state.tables.add(
+                            Table(
+                                fieldName = fieldName,
+                                isFieldNameWithArgs = isFieldNameWithArgs,
+
+                                isKindOfFieldTypeOptional = true,
+
+                                kindOfFieldType = kind0.toString(),
+                                fieldType = elementInfields.type.name,
+                                typeName = elementInTypes.name,
+
+                                enumValues = listOf(),
+                                unionTypes = listOf(),
+                                interfaceTypes = listOf()
+                            )
```

Comment:
i guess these `listOf()` could be good option for default values in `Table`, isn't it?

Reply:
Done